src/FE/src/app/pages/projects-page/projects-page.component.ts

```diff
@@ -162,15 +163,27 @@ export class ProjectsPageComponent implements AfterViewInit, OnInit {
   }
 
   async onDelete() {
-    const result = await this.projectService.deleteProject(this.selectedRow?.projectId!);
-    if (result) {
-      this.datasource.data = this.datasource.data.filter((project: Project) => project.projectId !== this.selectedRow?.projectId);
-      this.datasource._updateChangeSubscription();
-      this.updateTableFooterData();
-      this.snackbarService.openSnackBar(this.translateService.instant('SUCCESS_PROJECT_DELETED'), true);
-    } else {
-      this.snackbarService.openSnackBar(this.translateService.instant('ERROR_PROJECT_DELETED'), false);
-    }
+    const dialogRef = this.dialog.open(ConfirmDialogComponent, {
+      data: {
+        title: this.translateService.instant('CONFIRM_DELETE_PROJECT_TITLE'),
+        message: this.translateService.instant('CONFIRM_DELETE_MESSAGE') + this.selectedRow?.projectName + '?'
+      },
+      maxWidth: '15vw',
+      minWidth: '15vw',
+    });
+    dialogRef.afterClosed().subscribe(async (confirmed: boolean) => {
+      if (confirmed) {
+        const result = await this.projectService.deleteProject(this.selectedRow?.projectId!);
+        if (result) {
+          this.datasource.data = this.datasource.data.filter((project: Project) => project.projectId !== this.selectedRow?.projectId);
```

Comment:
kunne denne deles i to linjer? ved godt den var lang før, men nu hvor vi er forbi her